subsys/bluetooth/host/Kconfig

```diff
@@ -158,7 +158,7 @@ config BT_ACL_RX_COUNT
 	int "Number of incoming ACL data buffers"
 	default 6
 	default BT_CTLR_RX_BUFFERS if BT_CTLR
-	range 2 64
+	range 1 64
```

Comment:
Thinking aloud,
- I agree with the commit message,
- on combined builds ACL flow control is not used, this change has no effect, LE SC should work as before.
- builds without controller use 6 buffers, hence this change has no effect, LE SC should work as before.
- on combined builds, if ACL flow control is explicitly enabled by user, then he should take care to increase BT_ACL_RX_COUNT as necessary (like 2 when LE SC supported).